### ISSUE TITLE
junctions track display mode depends on loaded file name 

### ISSUE THREAD

**Issue by whtns** [igvteam/igv#1355]
I have generated bed files containing junction read counts which are displayed as sashimi plots in igv v2.15.4 when filenames end in "junctions.bed". If filenames do not end in "junctions.bed" they are displayed as typical genomic feature bed files. I was very surprised to see otherwise identical files displayed in different ways. I've not found documentation for this behavior or settings to alter display of junction bed files. See attached image

![image](https://github.com/igvteam/igv/assets/6510323/4048d45e-fe54-428c-a288-80cc1d124c17)

**Comment by jrobinso** [comment]
That's correct, this naming convention was adopted from "cuff links" where junctions.bed was originally defined.   There is nothing in the file.  You can also use a `track` line and set `graphType=junctions` as shown below. 

```
track name="TopHat junctions" description="TopHat junctions" graphType=junctions
```

**Comment by whtns** [comment]
Thank you! it's helpful to know about the graphType flag

**Comment by jrobinso** [comment]
I'm leaving this open as a documentation bug.
